lib/generators/i18n/active_record/install_generator.rb

```diff
@@ -19,7 +19,7 @@ def copy_initializer
         end
 
         def create_migrations
-          migration_template 'migration.rb.erb', 'db/migrate/create_translations.rb'
+          migration_template 'migration.rb.erb', 'db/migrate/create_#{table_name}.rb'
```

Comment:
Sorry, one more thing
```suggestion
          migration_template 'migration.rb.erb', "db/migrate/create_#{table_name}.rb"
```

Reply:
Sorry I copied it directly should have run tests or something afterwards.